### ISSUE TITLE
Fix Unique_Title metric descriptions

### ISSUE THREAD

**Issue by berndoberknapp** [Project-Counter/cop5#8]
The descriptions for Unique_Title_Investigations and Unique_Title_Requests in Table 3.s and Appendix A list both books and journals as examples of titles. Unique_Title metrics are only applicable to Data_Type Book, so this has to be fixed.

**Comment by tashalouizamc** [comment]
Suggest rewording as follows

The descriptions for Unique_Title_Investigations and Unique_Title_Requests in Table 3.s and Appendix A list both Books and Journals as examples of titles. Unique_Title metrics are only applicable to Data_Type Book, so this has to be fixed.

**Comment by berndoberknapp** [comment]
I wouldn't change the case for "books and journals", since this is a reference to 5.0.1 which uses lowercase because the context is titles, not Data_Types.
